t/Renard/Curie/Component/ScrollZoom.t

```diff
@@ -13,23 +13,24 @@ no strict;
 
 my $cairo_doc = CurieTestHelper->create_cairo_document;
 
-fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app) {
+fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app, (Int) $direction, (Int) $delta) {
 	my $event = Gtk3::Gdk::Event->new('scroll');
 	$event->state( 'control-mask' );
 	$event->direction('down');
 	$app->page_document_component->signal_emit( 'scroll_event' => $event );
```

Comment:
You are emitting the signal to the page drawing area, but it is not the page drawing area that handles this event. Look at which component is connected to the 'scroll-event' signal.